.travis.yml

```diff
@@ -1,9 +1,6 @@
 language: php
 php:
-  - 5.4
-  - 5.5
-  - 5.6
-  - hhvm
+  - '7.2'
```

Comment:
Is 7.2 a must?

Reply:
Nope, something was wrong with travis build and it was complaining about missing packages.
Support for PHP 7.1 will end in December 2019